sway/ipc-json.c

```diff
@@ -203,7 +221,7 @@ static void ipc_json_describe_workspace(struct sway_workspace *workspace,
 	for (int i = 0; i < workspace->floating->length; ++i) {
 		struct sway_container *floater = workspace->floating->items[i];
 		json_object_array_add(floating_array,
-				ipc_json_describe_node_recursive(&floater->node));
+				ipc_json_describe_node_recursive(&floater->node, &ipc_json_sway_descriptors));
```

Comment:
Style: 80 char limit

Reply:
Mmh, I thought going over 80 was okay as long as the line was readable and still above the 100 columns hard limit, but I just realized that the column size of a tab isn't defined in the contribution guidelines.

I'll fix this, but does a tab count for 8 columns, or 4 columns?

Reply:
4 cols. You can install an [editorconfig](http://editorconfig.org/) plugin for your text editor if it doesn't support it already.